<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8" />

<title>Benjamin L. Moore | Bioinformatics PhD</title>
<style type="text/css">
@import url("http://fonts.googleapis.com/css?family=OldStandardTT");@import url("http://fonts.googleapis.com/css?family=OpenSans");.container:before,.container:after{content:"";display:table}.container:after{clear:both}.container{zoom:1}body{font-family:'Open Sans','Myriad Pro','Myriad',sans-serif;color:#1d1313;font-weight:300;margin:0}h1,h2,h3,h4,ul dl dt{font-family:'Old Standard TT',serif}.container{margin:0 auto;padding:0 0 40px 0;background:#fff;border:solid #d22042;border-width:2px 0 0 0;text-align:left}.resume{position:relative;padding:10px 20px}blockquote a{color:#1d1313;-webkit-transition:color 0.3s linear;-moz-transition:color 0.3s linear;-ms-transition:color 0.3s linear;-o-transition:color 0.3s linear;transition:color 0.3s linear}blockquote a:hover{color:#d22042}a{color:#d22042}a[href$='.pdf']{display:inline-block;background:#1d1313;color:#fff;padding:6px 12px;margin-bottom:6px;text-decoration:none}blockquote{margin:0;padding:0;line-height:1.4em;text-align:right}hr:nth-of-type(1){margin-top:32px}hr{display:block;position:relative;padding:0;margin:18px auto;width:100%;clear:both;border:none;border-top:1px solid #ccc;font-size:1px;line-height:0;overflow:visible;page-break-after:avoid}h1{margin:0;padding:0;font-size:36px;letter-spacing:-1px;font-weight:normal}h2{margin:0;padding:0;font-size:18px;font-style:italic;letter-spacing:-1px;font-weight:normal}h3{margin:0;padding:0 0 0.5em;font-size:150%;font-style:italic;font-weight:normal}p{margin:0.6em 0 16px;padding:0;display:block;font-size:104%;line-height:24px}ul{margin:0;padding:0;list-style:none}ul li{margin:0;padding:0}ul li p{margin:1em 0.5em;line-height:120%}ul dl{margin:0.3em 0 0;padding:0;width:100%}ul dl dt{font-size:100%}ul dl dd{margin:0 0 1em;padding:0 2em 0 0;font-size:0.8em;line-height:1.5em}ol{margin:0;padding:0 0 0.75em;width:84%;display:inline-block}ol li{margin:0 0 0 1em;padding:0;width:100%;float:left;list-style:none;line-height:24px;font-size:14px}dl{display:inline-block;width:75%;margin:0;padding:0}dl dt{margin:0;padding:0;font-size:140%}dl dd{margin:0 0 1.5em;padding:0;font-size:80%;line-height:1.4em}dl strong{display:block}dl em{display:block;font-size:110%;margin:0.15em 0 0.5em;font-style:bold}#footer{display:none}#footer + p{width:100%;font-size:11px;text-align:center}@media screen{body{padding:0}.container{width:100%}}@media (min-width:1000px){.container{width:100%}.resume{margin:0 auto;padding:40px 50px;width:912px}blockquote{top:40px;right:50px;position:absolute}h1{margin-top:0;font-size:44px;text-transform:uppercase;letter-spacing:3px;font-weight:normal}h2{text-transform:uppercase;font-style:italic;letter-spacing:2px;font-weight:normal}h3{float:left;width:16%}h3 ~ p{float:left;margin-left:16%}h3 + p{margin-left:inherit;float:left;width:84%}ul li{width:28%;float:left}ul dl dt{font-size:122%;font-weight:normal;margin-bottom:0.75em}ul dl dd{padding:0 4em 0 0}ol{float:left;margin:0.6em 0 0}ol li{width:33%;margin:0}ol li:nth-child(3n){width:34%}ol li:nth-child(1),ol li:nth-child(2),ol li:nth-child(3){border-top:none}dl{margin:0.5em 0 0}dl strong{float:right;margin-top:-2em}dl em{font-size:130%;font-style:normal}}@media (max-width:1000px),print{blockquote{top:25px;right:50px;position:absolute}h1{margin-top:0.5em}ol{margin:0 0 0 2em;padding-bottom:0;width:100%}ol li{width:33%;margin:0}ol li:nth-child(1),ol li:nth-child(2){border-top:none}ul li{width:33%;float:left}p{margin-left:25px;font-size:14px;line-height:20px}p strong,p em{font-size:17px;line-height:24px}}@media screen and (max-width:625px){ul li{width:100%}ul br:nth-of-type(1){display:none}ul a{line-height:130%}blockquote{float:right;position:static}p{margin-left:5px;font-size:12px;line-height:16px}p strong,p em{font-size:14px;line-height:20px}ol li{width:50%;margin:0;font-size:12px}}@media print{.container{border-top:none}.not-printed{display:none}a{text-decoration:none;color:#000}.page-break{page-break-before:always;padding-top:20px}}@page{size:auto;margin:10mm}
</style>
  <style>
    a[href$='.pdf'] {
      display: none;
    }
  </style>
</head>

<body class="">

<div class="container">
    <div class="resume">
        <h1 id="benjamin-l-moore">Benjamin L. Moore</h1>
<h2 id="bioinformatics-phd">Bioinformatics PhD</h2>
<blockquote>
<p><a href="mailto:ben@blm.io">ben@blm.io</a><br>+44 (0)7854 212484</p>
</blockquote>
<hr>
<h3 id="education">Education</h3>
<p><strong>PhD Bioinformatics</strong> (2012-15) <em>University of Edinburgh</em><br>  During my PhD I analysed a diverse array of publicly-available biological datasets to uncover new insights into the epigenetics underlying higher-order chromatin stucture. I integrated Hi-C and other chromosome conformation capture experimental results with... My PhD was supervised by Colin A. Semple and Stuart Aitken in the MRC Human Genetics Unit, IGMM.<br><strong>MSc Bioinformatics and theoretical systems biology</strong> (2011-12) <em>Imperial College London</em>, Distinction<br>  My MSc included both taught (statistics, programming and biology) components, as well as three research projects. These included a software engineering project where I helped develop a stochastic Petri net library in Python with Prof. Michael Stumpf (<a href="https://github.com/sputnikpetrinets">Sputnik</a>), and another project (since <a href="https://dx.doi.org/10.1002/jcc.23330">published</a>) in which I applied Guassian mixture modelling to a long-standing problem in structural bioinformatics.<br><strong>BSc Biology</strong> (2008-11) <em>University of York</em>, First class honours<br>  My undergradute provided a broad education in biological sciences. My dissertation was a bioinformatics project in Prof. Peter Young&#39;s lab, looking at the evolution of toxin-antitoxin systems in <em>Rhizobia</em> genomes.</p>
<hr>
<h3 id="publications">Publications</h3>
<p>Integrative modelling reveals the principles of multi-scale chromatin boundary formation in human nuclear organization. (2015) <strong>Moore, BL</strong> <em>et al.</em> (Submitted).<br>High–quality protein backbone reconstruction from alpha carbons using Gaussian mixture models. (2013) <strong>Moore, BL</strong> <em>et al.</em> Journal of Computational Chemistry 34 (22), 1881-1889. doi:<a href="https://dx.doi.org/10.1002/jcc.23330">10.1002/jcc.23330</a><br>Rfam: Wikipedia, clans and the “decimal” release. (2011) PP Gardner, J Daub, J Tate, <strong>Moore, BL</strong> <em>et al.</em> Nucleic acids research 39 (S1), D141-D145. doi:<a href="https://dx.doi.org/10.1093/nar/gkq1129">10.1093/nar/gkq1129</a></p>
<hr>
<h3 id="placements">Placements</h3>
<p><strong>Wellcome Trust Sanger Institute</strong>, Cambridge.<br>During my undergraduate degree I spent two summers working in Alex Bateman&#39;s group on the <a href="http://rfam.xfam.org/">Rfam</a> database and supporting materials.</p>
<hr>
<h3 id="presentations">Presentations</h3>
<ul>
<li><strong>Talks</strong><br>4<sup>th</sup> Edinburgh bioinformatics meeting<br>EdinbR: The Edinburgh R usergroup (various)<br>Edinburgh Psychology R-users<br>Several research talks within the institute</li>
<li><strong>Posters</strong><br>Integrative modelling of higher order chromatin structure. <em>Genome Informatics</em> 2013 CSHL, NY, USA; <em>Keystone Epigenomics</em> 2015 Colorado, USA.<br>Unravelling higher order chromatin structure. <em>EpiGeneSys</em> 2013, Cambridge, UK.</li>
<li><strong>Tutorials</strong><br>WikiProject Computational Biology, ISMB 2014, Boston USA with Daniel Mietchen (half day), with grant support from the Wikimedia Foundation and ISCB.</li>
</ul>
<p><br clear="all" /><br /></p>
<hr>
<h3 id="technical">Technical</h3>
<ol>
<li>R and Bioconductor</li>
<li>Python and Biopython</li>
<li>LaTeX</li>
<li>Git / Mercurial</li>
<li>UNIX</li>
<li>Bash</li>
<li>Emacs</li>
<li>Inkscape</li>
<li>Mediawiki</li>
<li>Grid computing (SGE / PBS)</li>
<li>HTML / CSS / Javascript</li>
</ol>
<hr>
<h3 id="projects">Projects</h3>
<ul>
<li><p><strong>EdinbR</strong><br>  I co-founded a usergroup for the R programming language in Edinburgh called EdinbR. I help to organise our monthly meetings and run our website: <a href="http://edinbr.org">edinbr.org</a>. Our meetings attract 30-50 statisticians, data scientists and developers for talks and discussion about the R language and its applications.</p>
</li>
<li><p><strong>Summer Data Challenge</strong><br>  I entered Imperial College&#39;s <a href="http://summerdatachallenge.com/">Summer Data Challenge</a> competition, where entrants analysed a given dataset and then proposed a startup idea based on their results. I developed a modelling technique to select housing areas for investment and was awarded third place: £2,000 and startup support from Imperial Create Lab.</p>
</li>
<li><p><strong>Blog</strong><br>  I author a data science blog at <a href="http://blm.io">blm.io</a> where I apply R and Python to open datasets and write-up any interesting results. My posts have been picked up by sites including FiveThirtyEight, BuzzFeed, AVclub, io9 and more, and also led to being an invited blogger at the Huffington Post.</p>
</li>
</ul>
<hr>
<h3 id="awards">Awards</h3>
<p>MRC Capacity building scholarship (fees + enhanced stipend) for my PhD<br>MRC Full postgraduate scholarship (fees + stipend) for MSc<br><a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003242">ISCB computational biology Wikipedia competition</a>: second place prizewinner<br><a href="http://summerdatachallenge.com">Summer data challenge</a>: third place prizewinner</p>
<hr>
<h3 id="user-profiles">User profiles</h3>
<ol>
<li>Github: <a href="https://github.com/blmoore">blmoore</a></li>
<li>Twitter: <a href="https://twitter.com/benjaminlmoore">@benjaminlmoore</a></li>
<li>Wikipedia: <a href="https://en.wikipedia.org/wiki/User:Ben_Moore">Ben Moore</a></li>
<li>Domains: <a href="http://blm.io">blm.io</a>, <a href="http://edinbr.org">edinbr.org</a></li>
<li>Biostars: <a href="https://www.biostars.org/u/6239/">Ben</a></li>
<li>StackOverflow: <a href="http://stackoverflow.com/users/1274516/blmoore">blmoore</a></li>
</ol>
<hr>
<h3 id="references">References</h3>
<p>Available on request.</p>

    </div>
</div>

</body>
</html>